### DISCUSSION TITLE
体验了下，谷歌浏览器和EDGE 都试了有时候会无响应，编辑器好评

### DISCUSSION THREAD

**Discussion by LeannaClaire** [halo-dev/halo#3014] [General]
### 你当前使用的版本

2.0

### 描述一下此特性

Sakura主题仿Akina  彼岸也是仿的，这两个都挺好看，演示站是真的有点卡，不知道是服务器差了还是咋的，两个浏览器时不时的就无响应。

编辑器好评很好用，但是到发布文章->文章设置，我是真没见过这么蛋teng的发布设置，强制手动设置别名，不能留空自动生成别名，不能自动别名，给个自增的ID文章地址也行啊。

![QQ截图20221221004842](https://user-images.githubusercontent.com/34939681/208721078-a90d3100-1695-4b69-8fca-4006555196bb.png)

发布设置标签问题？

发布文章的时候，不能手动输入标签，只能选择现有的标签，这是咋想的？我文章都没写好，我咋想的到提前去把这篇文章内容当前要用到的标签设置好，看下图发布文章手动输入标签的选择。（多爽）
![QQ截图20221221005137](https://user-images.githubusercontent.com/34939681/208721875-9d18173c-e10c-458c-8bc3-3e0eabfea05a.png)

发布设置 常规和高级问题？
![QQ截图20221221005351](https://user-images.githubusercontent.com/34939681/208722225-447fb5dc-f8b2-4826-96a9-eb4c1991264c.png)

发布设置其他杂项移到高级设置是没问题，但是你们把封面移到高级设置你们去是要干嘛，你们看看我说的第一行那个主题那个不需要封面，我发布文章的时候还得再去点一下高级设置，然后再传图。这是闲步骤少了么？

如果你说封面可以自动提取内容图片当封面，那我要这封面功能干啥？

文档就瞄了一眼文章页面的PostVo

"creationTimestamp": "2022-11-20T12:45:43.888Z",    // 创建时间
"publishTime": "2022-11-20T13:06:38.505Z",          // 发布时间

当中的这个发布时间，是每次修改文章的时间，还是第一次发布的时间，这个没理解。问这个是制作结构化数据用到到，如下图：

![QQ截图20221221010356](https://user-images.githubusercontent.com/34939681/208724294-8fbf19ba-ce90-4198-bc5e-56bab37e8408.png)

![QQ截图20221221004510](https://user-images.githubusercontent.com/34939681/208720885-523f5679-e2d7-4fde-8fe0-d0f1bb05aecf.png)

**Comment by ruibaby** [comment]
感谢反馈，但你上面提到的问题建议每一项单独提交 issue，方便我们跟踪。

相关 issue：https://github.com/halo-dev/halo/issues/2859 https://github.com/halo-dev/halo/issues/2670
